pom.xml

```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>javacc21</groupId>
+	<artifactId>javacc</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<description>JavaCC21 ....</description>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+	<url>https://github.com/javacc21/javacc21</url>
+	<licenses>
+		<license>
+			<name>TODO</name>
+			<url>TODO</url>

```

Comment:
What is the license URL?

Reply:
How about [this?](https://javacc.com/LICENSE.html) (I just put that up there.)